src/figuresClasses.ts

```diff
@@ -1,19 +1,104 @@
+export type Shape = 'triangle' | 'circle' | 'rectangle';
+export type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+
+  getArea(): number;
+}
 
+export function roundArea(digit: number): number {
+  return Math.floor(digit * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape;
+
+  color: Color;
+
+  a: number;
+
+  b: number;
 
+  c: number;
```

Comment:
```suggestion
  shape: Shape = 'triangle';
```

Reply:
not quite like that, better use enum values, and use it everywhere
```suggestion
  shape: Shape = Shape.Triangle;
```